pkg/cmd/roachtest/tests/versionupgrade.go

```diff
@@ -111,6 +111,7 @@ func runVersionUpgrade(ctx context.Context, t test.Test, c cluster.Cluster) {
 	mvt := mixedversion.NewTest(
 		testCtx, t, t.L(), c, c.All(),
 		mixedversion.AlwaysUseFixtures, mixedversion.AlwaysUseLatestPredecessors,
+		mixedversion.NumUpgrades(1),
```

Comment:
Could we only pass this option in the `if c.IsLocal()` branch above? I think it would still be useful to run longer versions of this test during the nightly runs.